classes/PodsAPI.php

```diff
@@ -1750,11 +1753,10 @@ public function save_pod( $params, $sanitized = false, $db = true ) {
 					return pods_error( sprintf( __( 'Pod %s already exists', 'pods' ), $params->name ), $this );
 				}
 			}
-		} elseif ( in_array( $params->name, array(
-				'order',
-				'orderby',
-				'post_type'
-			) ) && 'post_type' === pods_var( 'type', $params ) ) {
+		} elseif (
+			in_array( $params->name, pods_reserved_keywords(), true )
+			&& 'post_type' === pods_v( 'type', $params )
```

Comment:
@sc0ttkclark Why is there a check for the Pod type? Aren't these keywords reserved for all types?

Reply:
Technically it’s for post types and taxonomies only based on WP codex, right?

Reply:
It looks like it. So I guess taxonomies should get the same validation right?

Reply:
Yes taxonomies too

Reply:
Done in the latest commit